101-webapp-linux-sonarqube-azuresql/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "siteName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of Azure Web App"
+            }
+        },
+        "sonarqubeImageVersion": {
+            "type": "string",
+            "defaultValue": "latest",
```

Comment:
add some allowedValues here for recent versions

Reply:
Done.

There are a large number of different image versions available, so I've chosen the most likely current ones. The image versions increase regularly though. I've also changed the Default image to the LTS one instead of Latest as I think this is more likely to be a safer default.